menas/ui/components/schema/schemaMain.controller.js

```diff
@@ -104,14 +104,17 @@ sap.ui.controller("components.schema.schemaMain", {
 	},
 
 	resetNewSchemaValueState : function() {
-		sap.ui.getCore().byId("newSchemaName").setValueState(sap.ui.core.ValueState.None)
-		sap.ui.getCore().byId("newSchemaName").setValueStateText("")
+		sap.ui.getCore().byId("newSchemaName").setValueState(sap.ui.core.ValueState.None);
+		sap.ui.getCore().byId("newSchemaName").setValueStateText("");
+
+    this.byId("fileUploader").setValueState(sap.ui.core.ValueState.None);
+    this.byId("fileUploader").setValueStateText("");
```

Comment:
We should try to move towards controller-bound controls (i.e., `this.byId()`), rather than using global access. Not sure if this will work in all cases, but I don't know when it wouldn't, so let's try and find out.

Reply:
This also applies for just where the task requires the code to change.

Reply:
Since its view descendant and cant access the setValueState property, we need to stick to this.byid("fileUploader").[more info here](https://stackoverflow.com/questions/48639302/difference-between-this-getview-byid-this-byid-and-sap-ui-getcore-byid?noredirect=1&lq=1)

Reply:
resolved.